### PR TITLE
PHEE-387 Create fineract stub APIs in AMS_mifos for successful response.

### PR DESCRIPTION
**Pull request by logoutdhaval**
## Description

[PHEE-387 Create fineract stub APIs in AMS_mifos for successful response.](https://fynarfin.atlassian.net/browse/PHEE-387?atlOrigin=eyJpIjoiZmQyMDcxMDgxOTFkNGI3YTlkOTBkMzc3ODU0OWVjY2UiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

File: src/main/java/org/mifos/connector/ams/interop/AmsCommonService.java [hunk 1]
```diff
@@ -26,7 +26,7 @@
 import static org.mifos.connector.ams.zeebe.ZeebeVariables.TENANT_ID;
 
 @Component
-@ConditionalOnExpression("${ams.local.enabled}")
```
Comment (fynmanoj) [inline src/main/java/org/mifos/connector/ams/interop/AmsCommonService.java:29]: ams_local_enable = true,  then go to fineract
ams_local_enable = false, then go to mock api

File: src/main/resources/application-fin12.yml [hunk 2]
```diff
@@ -40,6 +40,18 @@ ams:
       host: http://localhost:8443
       repayment-path: /fineract-provider/api/v1/interoperation/transactions/{accountNumber}/loanrepayment
 
-
+mock-service:
+  local:
+    loan:
+      host: http://localhost:7070
```
Comment (fynmanoj) [inline src/main/resources/application-fin12.yml:46]: why is the port 7070? are we exposing it out?

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -30,3 +30,22 @@ ams:
...
+
+springfox:
+  documentation:
+    open-api:
+      v3:
+        path: /api-docs
+server:
+  servlet:
+    contextPath: /fineract
+  port: 7070
```
Comment (fynmanoj) [inline src/main/resources/application.yml:51]: is this an additional port?

**Review by fynmanoj — APPROVED** [review]

File: src/main/resources/application-fin12.yml (hunk 2 above)
Comment (logoutdhaval) [reply src/main/resources/application-fin12.yml:46]: 7070 port for mock server.

File: src/main/resources/application.yml (hunk 3 above)
Comment (logoutdhaval) [reply src/main/resources/application.yml:51]: yes, for mock server.

**Pull request merged**